app/javascript/utils/react-bootloader.tsx

```diff
@@ -68,10 +68,14 @@ function initEventListeners() {
   // render those when the frame changes. We want the CSS to load BEFORE
   // then HTML renders, so we get any stylesheets downloaded and THEN render
   // continnue processing the frame render.
-  document.addEventListener('turbo:before-frame-render', (e) => {
-    const hrefs = Array.from(event.detail.newFrame.getElementsByTagName('link'))
-      .filter((el) => el.getAttribute('rel') == 'stylesheet')
-      .map((el) => el.getAttribute('href'))
+  document.addEventListener('turbo:before-frame-render', (e: Event) => {
+    if (!(e instanceof CustomEvent)) return
+
+    const links: HTMLCollectionOf<HTMLLinkElement> =
+      e.detail.newFrame.getElementsByTagName('link')
+    const hrefs = Array.from(links)
+      .filter((link) => link.getAttribute('rel') == 'stylesheet')
+      .map((link) => link.getAttribute('href'))
```

Comment:
Would it perhaps make sense to use `querySelectorAll` to combine the selecting of the elements and the filtering?

Reply:
Thanks for the suggestions, pushed.